# -*- coding: utf-8 -*-
PUBS_FIELD_COUNT = 6 

#Names
#Title
#Journal
#Year
#URL
#Research Project

PUBS = """

Makova KD, Patton JC.
Increased yield of tri- and tetranucleotide microsatellite markers from unenriched small-insert libraries
BioTechniques
1998
http://www.ncbi.nlm.nih.gov/pubmed/9454948
Mc

Makova KD, Patton JC, Krysanov EYu, Chesser RK, Baker RJ.
Microsatellite markers in wood mouse and striped field mouse (genus Apodemus)
Molecular Ecology
1998
http://onlinelibrary.wiley.com/doi/10.1111/j.1365-294X.1998.00315.x/abstract
Mc

Baker RJ, Makova KD, Chesser RK.
Microsatellites indicate high frequency of multiple paternity in Apodemus (Rodentia)
Molecular Ecology
1999
http://onlinelibrary.wiley.com/doi/10.1046/j.1365-294X.1999.00541.x/abstract
Mc

Nekrutenko A, Makova KD, Chesser RK, Baker RJ.
Representational Difference Analysis to distinguish cryptic species
Molecular Ecology
1999
http://onlinelibrary.wiley.com/doi/10.1046/j.1365-294X.1999.00682_1.x/abstract
Mc

Nekrutenko A, Makova KD, Baker RJ.
Isolation of binary species-specific PCR-based markers and their value for systematic applications
Gene
2000
http://eurekamag.com/research/010/893/010893188.php
Mc

Makova KD, Nekrutenko A, Baker RJ.
Evolution of Microsatellite Alleles in Four species of mice (genus Apodemus)
Journal of Molecular Evolution
2000
http://www.ncbi.nlm.nih.gov/pubmed/10948273
Mc

Longmire JL, Roach JL, Maltbie M, White PS, Tatum OL, Makova KD, Hahn DC.
Tetranucleotide microsatellite markers for the brown-headed cowbird (Molothrus ater)
Journal of Avian Biology
2001
http://onlinelibrary.wiley.com/doi/10.1034/j.1600-048X.2001.320111.x/abstract
Mc

Liu JC, Makova KD, Adkins RM, Gibson S, Li WH.
Periods of rapid evolution of growth hormone in primates and emergence of species specificity of growth hormone receptor
Molecular Biology and Evolution
2001
http://mbe.oxfordjournals.org/content/18/6/945.long
R

Makova KD, Ramsay M, Jenkins T, Li WH.
Human DNA sequence variation in a 6.6-kilobase region containing the melanocortin 1 receptor promoter
Genetics
2001
http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1461732/
R

Nekrutenko A, Makova KD, Li WH.
The KA/KS ratio test for assessing the protein-coding potential of genomic regions: An empirical and simulation study
Genome Research
2002
http://www.ncbi.nlm.nih.gov/pmc/articles/PMC155263/
R

Makova KM, Li WH.
Strong male-driven evolution of DNA sequences in humans and apes
Nature
2002
http://www.nature.com/nature/journal/v416/n6881/full/416624a.html
S, R

Mu J, Duan J, Makova KD, Joy DA, Huynh CQ, Branch OH, Li WH, Su XZ.
Chromosome-wide SNPs reveal an ancient origin for Plasmodium falciparum
Nature
2002
http://www.nature.com/nature/journal/v418/n6895/full/nature00836.html
R

Li WH, Yi S, Makova K.
Male-driven evolution
Current Opinions in Genetics and Development.
2002
http://www.sciencedirect.com/science/article/pii/S0959437X02003544
S, R

Li WH, Makova KM.
Domain duplication and gene elongation
Encyclopedia of the Human Genome (Nature Publishing Group)
2003
http://www.els.net/WileyCDA/ElsArticle/refId-a0005097.html
R

John PR, Makova K, Li WH, Jenkins T, Ramsay M.
DNA polymorphism and selection at the melanocortin-1 receptor gene in normally pigmented southern African individuals
Ann N Y Acad Sci
2003
http://onlinelibrary.wiley.com/doi/10.1111/j.1749-6632.2003.tb03193.x/abstract
R

Makova KD, Li WH.
Divergence in the spatial pattern of gene expression between human duplicate genes
Genome Research
2003
http://genome.cshlp.org/content/13/7/1638.full
R

International Chicken Genome Sequencing Consortium (K. D. Makova - member)
Sequence and comparitive analysis of the chicken genome provide unique perspectives on vertebrate evolution
Nature
2004
http://www.nature.com/nature/journal/v432/n7018/full/nature03154.html
R, S

Rat Genome Sequencing Project Consortium (K. D. Makova - member)
Evolution of the Mammalian Genome: sequence of the genome of the brown Norwegian Rat
Nature
2004
http://www.nature.com/nature/journal/v428/n6982/abs/nature02426.html
R, S

Miller W, Makova KM, Nekrutenko A, Hardison RC.
Comparitive Genomics
Annual Review of Genomics and Human Genetics
2004
http://www.annualreviews.org/doi/full/10.1146/annurev.genom.5.061903.180057?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%3dpubmed&
?

Makova KD, Yang S, Chiaromonte F.
Indels are male-biased too: a whole genome analysis in rodents
Genome Research
2004
http://genome.cshlp.org/cgi/pmidlookup?view=long&pmid=15059997
R

Nekrutenko A, Wadhawan S, Goetting-Minesky P, Makova KD.
Oscillating Evolution of a Mammalian Locus with Overlapping Reading Frames: An XLas/ALEX Relay
PloS Genetics
2005
http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.0010018
R

The Chimpanzee Sequencing and Analysis Consortium (K. D. Makova - member)
Initial sequence of the chimpanzee genome and comparison with the human genome
Nature
2005
http://www.nature.com/nature/journal/v437/n7055/full/nature04072.html
R

Makova K, Norton H.
Worldwide polymorphism at the MC1R locus and normal pigmentation variation in humans
Peptides
2005
http://www.sciencedirect.com/science/article/pii/S0196978105002779
R

Taylor J, Tyekucheva S, Zody M, Chiaromonte F, Makova KD.
Strong and Weak Male Mutation Bias at Different Sites in the Primate Genomes: Insights from the Human-Chimpanzee Comparison
Molecular Biology and Evolution
2006
http://mbe.oxfordjournals.org/content/23/3/565.full.pdf
R, S

Chung WY, Albert R, Albert I, Nekrutenko A, Makova KD.
Rapid and asymmetric divergence of duplicate genes in the human gene coexpression network
BMC Bioinformatics
2006
http://www.biomedcentral.com/1471-2105/7/46
R

Goetting-Minesky MP, Makova KD.
Mammalian Male Mutation Bias: Impacts of Generation Time and Regional Variation in Substitution Rates
Journal of Molecular Evolution
2006
http://link.springer.com/article/10.1007%2Fs00239-005-0308-8
S, R

Carrel L, Park C, Tyekucheva S, Dunn J, Chiaromonte F, Makova KD.
Genomic environment predicts expression patterns on the human inactive X chromosome
PLoS Genet
2006
http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.0020151
S

Rhesus Macaque Genome Sequencing and Analysis Consortium (K. D. Makova - member)
Evolutionary and Biomedical Insights from the Rhesus Macaque Genome
Science
2007
http://www.sciencemag.org/content/316/5822/222
S

Blankenberg D, Taylor J, Schenck I, He J, Zhang Y, Ghent M, Veeraraghavan N, Albert I, Miller W, Makova KD, Hardison RC, Nekrutenko A.
A framework for collaborative analysis of ENCODE data: Making large-scale analyses biologist-friendly
Genome Research
2007
http://genome.cshlp.org/content/17/6/960.long
?

Kvikstad EM, Tyekucheva S, Chiaromonte F, Makova KD.
A macaque's-eye view of human insertions and deletions: differences in mechanisms
PLoS Computational Biology
2007
http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.0030176
R

Li WH, Makova KD
Molecular Clocks
Encyclopedia of Genetics (Academic Press)
2008
http://www.els.net/WileyCDA/ElsArticle/refId-a0005111.html
R

Kelkar YD, Tyekucheva S, Chiaromonte F, Makova KD.
The genome-wide determinants of human and chimpanzee microsatellite evolution
Genome Research
2008
http://genome.cshlp.org/content/18/1/30.long
Mc

Tyekucheva S, Makova KD, Karro JE, Hardison RC, Miller W, Chiaromonte F.
Human-macaque comparisons illuminate variation in neutral substitution rates
Genome Biology
2008
http://www.genomebiology.com/2008/9/4/R76
R

Lau AN, Peng L, Goto H, Chemnick L, Ryder OA, Makova KD.
Horse Domestication and Conservation Genetics of Przewalski's Horse Inferred from Sex Chromosomal and Autosomal Sequences
Molecular Biology and Evolution
2009
http://mbe.oxfordjournals.org/content/26/1/199.abstract
S

Goto H, Peng L, Makova KD.
Evolution of X-degenerate Y chromosome genes in greater apes: Conservation of gene content in human and gorilla, but not chimpanzee
Journal of Molecular Evolution
2009
http://link.springer.com/article/10.1007%2Fs00239-008-9189-y
S

Park C, Makova KD.
Coding region structural heterogeneity and turnover of transcription start sites contribute to divergence in expression between duplicate genes
Genome Biology 
2009
http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2687787/
R

Kvikstad EM, Chiaromonte F, Makova KM.
Ride the wavelet: a multi-scale analysis of genomic contexts flanking small insertions and deletions
Genome Research
2009
http://genome.cshlp.org/cgi/pmidlookup?view=long&pmid=19502380
R

Wilson MA, Makova KD.
Genomic analyses of sex chromosome evolution
Annual Reviews of Human Genetics and Genomics
2009
http://www.annualreviews.org/doi/full/10.1146/annurev-genom-082908-150105?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%3dpubmed
S

Wilson MA, Makova KD.
Evolution and survival on eutherian sex chromosomes
PLoS Genetics
2009
http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1000568
S

Schuster SC, Miller W, Ratan A, Tomsho LP, Giardine B, Kasson LR, Harris RS, Petersen DC, Zhao F, Qi J, Alkan C, Kidd JM, Sun Y, Drautz DI, Bouffard P, Muzny DM, Reid JG, Nazareth LV, Wang Q, Burhans R, Riemer C, Wittekindt NE, Moorjani P, Tindall EA, Danko CG, Teo WS, Buboltz AM, Zhang Z, Ma Q, Oosthuysen A, Steenkamp AW, Oostuisen H, Venter P, Gajewski J, Zhang Y, Pugh BF, Makova KD, Nekrutenko A, Mardis ER, Patterson N, Pringle TH, Chiaromonte F, Mullikin JC, Eichler EE, Hardison RC, Gibbs RA, Harkins TT, Hayes VM.
Complete Khoisan and Bantu genomes from southern Africa
Nature
2010
http://www.nature.com/nature/journal/v463/n7283/full/nature08795.html
?

Kvikstad EM, Makova KD.
The (r)evolution of LINE and SINE distribution in the primate genomes: Sex chromosomes are important
Genome Research
2010
http://genome.cshlp.org/cgi/pmidlookup?view=long&pmid=20219940
R

Park C, Carrel L, Makova KD.
Strong Purifying Selection at Genes Escaping X Chromosome Inactivation
Molecular Biology and Evolution
2010
http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2981488/
S

Kelkar YD, Strubczewski N, Hile SE, Chiaromonte F, Eckert KA, Makova KD.
What Is a Microsatellite: A Computational and Experimental Definition Based upon Repeat Mutational Behavior at A/T and GT/AC Repeats
Genome Biol. Evol.
2010
http://gbe.oxfordjournals.org/cgi/pmidlookup?view=long&pmid=20668018
Mc

Ananda G, Chiaromonte F, Makova KD.
A genome-wide view of mutation rate co-variation using multivariate analyses
Genome Biology
2011
http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3129677/
R

Wilson Sayres MA, Venditti C, Pagel M, Makova KD.
Do variations in substitution rates and male mutation bias correlate with life history traits? A study of 32 mammalian genomes
Evolution
2011
http://onlinelibrary.wiley.com/doi/10.1111/j.1558-5646.2011.01337.x/abstract
R, S

Goto H, Dickins B, Afgan E, Paul IM, Taylor J, Makova KD, Nekrutenko A.
Dynamics of mitochondrial heteroplasmy in three families investigated via a repeatable re-sequencing study
Genome Biology
2011
http://www.genomebiology.com/2011/12/6/r59
Mt

Lombard Z, Park C, Makova KD, Ramsay M.
A Computational Approach to Candidate Gene Prioritization for X-Linked Mental Retardation using Annotation-based Binary Filtering and Motif-Based Linear Discriminatory Analysis
Biology Direct
2011
http://www.biologydirect.com/content/6/1/30
S

Goto H, Ryder OA, Fisher AR, Schultz B, Kosakovsky Pond SL, Nekrutenko A, Makova KD.
A massively parallel sequencing approach uncovers ancient origins and high genetic variability of endangered Przewalski s horses
Genome Biology and Evolution
2011
http://gbe.oxfordjournals.org/cgi/pmidlookup?view=long&pmid=21803766
Mt

Afgan E, Baker D, Coraor N, Goto H, Paul IM, Makova KD, Nekrutenko A, Taylor J.
Harnessing cloud-computing for biomedical research with Galaxy Cloud
Nature Biotechnology
2011
http://www.nature.com/nbt/journal/v29/n11/abs/nbt.2028.html
Mt

Kelkar YD, Eckert KA, Chiaromonte F, Makova KD.
A matter of life or death: How microsatellites emerge in and vanish from the human genome
Genome Research
2011
http://genome.cshlp.org/cgi/pmidlookup?view=long&pmid=21994250
Mc

Wilson Sayres MA, Makova KD.
Genome analyses substantiate male mutation bias in many species
Bioessays
2011
http://onlinelibrary.wiley.com/doi/10.1002/bies.201100091/abstract
S

Wolfe K, Wayne ML, Makova K, Steen TY, Uyenoyama M, Takahata N.
SMBE Proposal to the Government of Japan
Molecular Biology and Evolution
2012
http://mbe.oxfordjournals.org/content/29/1/441.full
?

Fungtammasan A, Walsh E, Chiaromonte F, Eckert KA, Makova KD.
A Genome-wide Analysis of Common Fragile Sites: What Features Determine Chromosomal Instability in the Human Genome?
Genome Research
2012
http://genome.cshlp.org/cgi/pmidlookup?view=long&pmid=22456607
Mc, R

Wagstaff BJ, Hedges DJ, Derbes RS, Campos Sanchez R, Chiaromonte F, Makova KD, Roy-Engel AM.
Rescuing Alu: recovery of new inserts shows LINE-1 preserves Alu activity through A-tail expansion
PLoS Genetics
2012
http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1002842
R

Wilson Sayres MA, Makova KD.
Gene survival and death on the human Y chromosome
Molecular Biology and Evolution 
2013
http://mbe.oxfordjournals.org/cgi/pmidlookup?view=long&pmid=23223713
S

Ananda G, Walsh E, Jacob KD, Krasilnikova M, Eckert KA, Chiaromonte F, Makova KD.
Distinct mutational behaviors distinguish short tandem repeats from microsatellites in the human genome
Genome Biology and Evolution
2013
http://gbe.oxfordjournals.org/content/5/3/606/suppl/DC1
Mc

Baptiste BA, Ananda G, Strubczewski N, Lutzkanin A, Khoo SJ, Srikanth A, Kim N, Makova KD, Krasilnikova MM, Eckert KA.
Mature Microsatellites: Mechanisms Underlying Dinucleotide Microsatellite Mutational Biases in Human Cells
Genes, Genomes, and Genetics
2013
http://www.g3journal.org/cgi/pmidlookup?view=long&pmid=23450065
Mc

Montgomery SB, Goode DL, Kvikstad E, Albers CA, Zhang ZD, Mu XJ, Ananda G, Howie B, Karczewski KJ, Smith KS, Anaya V, Richardson R, Davis J; 1000 Genomes Project Consortium, MacArthur DG, Sidow A, Duret L, Gerstein M, Makova KD, Marchini J, McVean G, Lunter G.
The origin, evolution, and functional impact of short insertion-deletion variants identified in 179 human genomes
Genome Research
2013
http://genome.cshlp.org/content/early/2013/03/11/gr.148718.112.abstract
Mc

Steiner CC, Makova KD, Ryder OA.
Mitochondrial Genome: Clues about the Evolution of Extant Equids and Genomic Diversity of Horse Breeds, in Equine Genomics
(ed B. P. Chowdhary), Blackwell Publishing Ltd., Oxford, UK
2013
http://onlinelibrary.wiley.com/doi/10.1002/9781118522158.ch19/summary
Mt

Kuruppumullage Don P, Ananda G, Chiaromonte F, Makova KD.
Segmenting the human genome based on states of neutral genetic divergence
Proceedings of the National Academy of Sciences
2013
http://www.pnas.org/content/110/36/14699.long
R

Dickins B, Rebolledo-Jaramillo B, Su MS3, Paul IM, Blankenberg D, Stoler N, Makova KD, Nekrutenko A.
Controlling for contamination in re-sequencing studies with a reproducible web-based phylogenetic approach
BioTechniques
2014
http://www.biotechniques.com/BiotechniquesJournal/2014/March/Controlling-for-contamination-in-re-sequencing-studies-with-a-reproducible-web-based-phylogenetic-approach/biotechniques-350711.html
Mt

Kvikstad EM, Makova KD.
Rapid Evolution of Genes on the Human X-chromosome. 
Encyclopedia of Life Sciences (ELS)
2013
http://onlinelibrary.wiley.com/doi/10.1002/9780470015902.a0020858.pub2/abstract
S

Campos-Sánchez R, Kapusta A, Feschotte C, Chiaromonte F, Makova KD.
Genomic Landscape of Human, Bat, and Ex Vivo DNA Transposon Integrations
Molecular Biology and Evolution
2014
http://mbe.oxfordjournals.org/content/early/2014/04/22/molbev.msu138
R

McElhoe JA, Holland MM, Makova KD, Su MS, Paul IM, Baker CH, Faith SA, Young B.
Development and assessment of an optimized next-generation DNA sequencing approach for the mtgenome using the Illumina MiSeq
Forensic Science International: Genetics
2014
http://www.sciencedirect.com/science/article/pii/S1872497314000994
Mt

Ananda G, Hile SE, Breski A, Wang Y, Kelkar Y, Makova KD, Eckert KA.
Microsatellite Interruptions Stabilize Primate Genomes and Exist as Population-Specific Single Nucleotide Polymorphisms within Individual Human Genomes
PLoS Genet.
2014
http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1004498
Mc

Paul IM, Williams JS, Anzman-Frasca S, Beiler JS, Makova KD, Marini ME, Hess LB, Rzucidlo SE, Verdiglione N, Mindell JA, Birch LL.
The Intervention Nurses Start Infants Growing on Healthy Trajectories (INSIGHT) study
BMC pediatrics
2014
http://www.biomedcentral.com/1471-2431/14/184
O

Rebolledo-Jaramillo B, Su MS, Stoler N, McElhoe JA, Dickins B, Blankenberg D, Korneliussen TS, Chiaromonte F, Nielsen R, Holland MM, Paul IM, Nekrutenko A, Makova KD.
Maternal Age Effect and Severe Germline Bottleneck in the Inheritance of Human Mitochondrial DNA
PNAS
2014
http://www.pnas.org/content/111/43/15474.full
Mt

Chiaromonte F, Makova KD.
Using Statistics to Shed Light on the Dynamics of the Human Genome: A Review. Advances in Complex Data Modeling and Computational Methods in Statistics
Contributions to Statistics
2014
http://link.springer.com/chapter/10.1007%2F978-3-319-11149-0_5
R

Makova KD, Hardison RC.
The effects of chromatin organization on variation in mutation rates in the genome
Nature Reviews Genetics
2015
http://www.nature.com/nrg/journal/v16/n4/full/nrg3890.html
R

Fungtammasan A, Ananda G, Hile SE, Su MS, Sun C, Harris R, Medvedev P, Eckert K, Makova KD.
Accurate Typing of Short Tandem Repeats from Genome-wide Sequencing Data and its Applications
Genome Research
2015
http://genome.cshlp.org/content/early/2015/03/30/gr.185892.114
Mc

Tomaszkiewicz M, Rangavittal S, Cechova M, Campos Sanchez R, Fescemyer HW, Harris R, Ye D, O'Brien PC, Chikhi R, Ryder OA, Ferguson-Smith MA, Medvedev P, Makova KD.
A Time- and Cost-Effective Strategy to Sequence Mammalian Y Chromosomes: An Application to the de novo Assembly of Gorilla Y
Genome Research
2016
http://genome.cshlp.org/content/early/2016/02/26/gr.199448.115.abstract
S

Campos-Sánchez R, Cremona MA, Pini A, Chiaromonte, Makova KD.
Integration and Fixation Preferences of Human and Mouse Endogenous Retroviruses Uncovered with Functional Data Analysis
PLOS Computational Biology
2016
http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1004956
R

Fungtammasan A, Tomaszkiewicz M, Campos-Sánchez R, Eckert KA, DeGiorgio M, Makova KD.
Reverse Transcription Errors and RNA-DNA Differences at Short Tandem Repeats
Molecular Biology and Evolution
2016
http://mbe.oxfordjournals.org/content/early/2016/07/11/molbev.msw139.abstract?keytype=ref&ijkey=vL8Oo9g8zVeNL8F
Mc

Arbeithuber B, Makova KD, Tiemann-Boege I.
Artifactual mutations resulting from DNA lesions limit detection levels in ultrasensitive sequencing applications
DNA research
2016
http://dnaresearch.oxfordjournals.org/content/early/2016/07/27/dnares.dsw038.full.pdf?keytype=ref&ijkey=wzzYwZjQ8tWQAiV
Mt

Stoler N, Arbeithuber B, Guiblet W, Makova KD, Nekrutenko A.
Streamlined analysis of duplex sequencing data with Du Novo
Genome Biology
2016
http://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-1039-4
Mt

Tomaszkiewicz M, Medvedev P, Makova KD.
Y and W Chromosome Assemblies: Approaches and Discoveries
Trends in Genetics
2017
http://www.sciencedirect.com/science/article/pii/S0168952517300197
S

Baker RJ, Dickins B, Wickliffe JK, Khan FA, Gaschak S, Makova KD, Phillips CD. 
Elevated mitochondrial genome variation after 50 generations of radiation exposure in a wild rodent
Evolutionary Applications
2017
http://onlinelibrary.wiley.com/doi/10.1111/eva.12475/abstract
Mt

Rangavittal S, Harris R, Cechova M, Tomaszkiewicz M, Chikhi R, Makova KD, Medvedev P.
RecoverY : K-mer based read classification for Y-chromosome specific sequencing and assembly
bioRxiv
2017
https://www.biorxiv.org/content/early/2017/11/27/148114
S

Warris S, Schijlen E, van de Geest H,  Vegesna R, Hesselink T, te Lintel Hekkert B, Sanchez-Perez G, Medvedev P, Makova KD, de Ridder D
Correcting palindromes in long reads after whole-genome amplification
bioRxiv
2017
http://www.biorxiv.org/content/early/2017/08/09/173872
S

Wilton P, Zaidi A, Makova KD, Nielsen R.
A population phylogenetic view of mitochondrial heteroplasmy
bioRxiv
2017
https://www.biorxiv.org/content/early/2017/10/17/204479
Mt

Craig S, Blankenberg D, Parodi A, Paul I, Birch L, Savage J, Marini M, Stokes J, Nekrutenko A, Reimherr M, Chiaromonte F, Makova KD.
Infant Weight Gain Trajectories Linked To Oral Microbiome Composition
bioRxiv
2017
https://www.biorxiv.org/content/early/2017/10/24/208090
O

"""

def publications():
	lines = PUBS.splitlines()
	lines = map(lambda x: x.strip(), lines)
	lines = filter(None, lines)
	group = [iter(lines)] * PUBS_FIELD_COUNT
	list = zip(*group)
	list.reverse()
	return list


NEWS_FIELD_COUNT = 2

#Announcement
#Icone

NEWS = """

Read Rebeca's latest article in Molecular Biology and Evolution : Genomic Landscape of Human, Bat and Ex Vivo DNA Transposon Integrations
file

Congratulations to Sarah on winning the Penn State Clinical and Translational Science Institute Novel Methodologies in Health Research Award!
thumbs-up

Makova Lab graduate students Rebeca and Samarth awarded the Jeanette Ritter Mohnkern Graduate Student Scholarship in Biology and Huck Dissertation Research Award respectively
thumbs-up

Read about our latest research on the maternal age affect on human mtDNA in the October 28, 2014 edition of PNAS.
file

Monika Cechova (Michalovova) receives the CBIOS predoctoral training grant from the NIH's National Institute of General Medical Sciences !
thumbs-up

Congratulations to Wilfried Guiblet on winning the Huck Dissertation Research Award 2015!
thumbs-up

Congratulations to Marta Tomaszkiewicz for her 3rd place during the 8th Annual Postdoc Research Exhibition !
thumbs-up

Congratulations to Malika Laurence for her grant awarded by the Undergraduate Research Fund !
thumbs-up

Congratulations to the Gorilla Y Chromosome group for their paper being accepted in Genome Research !
thumbs-up

Congratulations to Monika Cechova (Michalovova) on winning the 2016 Dr. John Randall Shuman Troxell Memorial Scholarship in Biology!
thumbs-up

Congratulations to Sarah Craig and Marta Tomaszkiewicz for winning the 1st and 2nd poster awards at the Center for Medical Genomics Retreat!
thumbs-up

February 21, 2017 : Marta Tomaszkiewicz reviews W and Y chromosome assemblies in the April 2017 issue of Trends in Genetics.
file

April 27, 2017 : Congratulations to Danling for being the recipient of the 2017 Douglas and Regina Award for Research Achievement for her work in the lab on ampliconic gene copy numbers in the human Y chromosome!
thumbs-up

July 25, 2017 : Barbara Arbeithuber is awarded the Erwin Schrödinger fellowship. Congrats!
thumbs-up

August 17, 2017 : Bruce is runner-up in the MCIBS & Pathobiology retreat poster competition. Congratulations!
thumbs-up

October 31, 2017 : Di (Bruce) Chen's L1 project is selected for the "Future of Science Fund Scholarship" at the Mobile Genetic Elements meeting (2017). Congrats Bruce!
thumbs-up

November 23, 2017 : The Y-chromosome group's manuscript on read classification from Y-enriched data accepted for publication in Bioinformatics.
file

"""

def news():
	lines = NEWS.splitlines()
	lines = map(lambda x: x.strip(), lines)
	lines = filter(None, lines)
	group = [iter(lines)] * NEWS_FIELD_COUNT
	list = zip(*group)
	list.reverse()
	return list

#Name and Info

ALUMNI = """


Zaky Adam - Postdoctoral Scholar 
Guru Ananda - Postdoctoral Scholar 
Amanda Breski - Undergraduate student - Biology
Rebeca Campos-Sanchez - Ph.D student, Intercollege Graduate Program in Genetics
Benjamin Dickins - Postdoctoral Scholar 
Prabhani Kuruppumullage Don - Ph.D. student - Statistics 
Allison Fisher - Undergraduate student - Biology
Arkarachai Fungtammasan - Ph.D student - Bioinformatics and Genomics
Matthew Ghent - Undergraduate - Biochemistry 
Aparna Goel - Undergraduate student 
Paula Goetting - Senior research assistant 
Hiroki Goto - Postdoctoral Scholar 
Yogeshwar Kelkar - Ph.D. student - Biology 
Lydia Krasilnikova - Undergraduate student - Computer Science 
Erika Kvikstad - Ph.D. student - Genetics 
Allison Lau - Undergrad - Biology 
Malika Laurence - Undergraduate Student, Premedical-Medical Program
Ali Martinson - Undergraduate student - Biology 
Matthew Nielsen - Undergraduate student - Biology 
Chungoo Park - Ph.D. student - Biology 
Lei Peng - Undergrad - Biology 
Boris Rebolledo-Jaramillo - Ph.D. student - Bioinformatics and Genomics
Adil Sarwar - Ph.D. student - Computer Science
Marcia Shu-Wei Su - Postdoctoral Scholar 
Melissa A. Wilson Sayres - Ph.D. student - Integrative Biosciences 
Bryant Schultz - Undergraduate student - Biology 
Carolyn Rogers - Undergraduate Student - Program in Biology

"""

def alumni():
	lines = ALUMNI.splitlines()
	lines = map(lambda x: x.strip(), lines)
	lines = filter(None, lines)
	return lines




PEOPLE_FIELD_COUNT = 7

#Name
#Job
#Contact
#Description
#Picture
#Research Project
#URL to personal page

PEOPLE = """

Barbara Arbeithuber
Postdoctoral Scholar
bxa15@psu.edu
I am interested in the accurate measurement of rare mutations. My current project focuses on the inheritance of mitochondrial DNA; particularly, I want analyze de novo mutations in the germ line, as well as the inheritance of low-level variants.
Barbara.png
Mt
NA

Sarah J. Carnahan Craig
Postdoctoral Scholar
sarah@bx.psu.edu	Lab : (814) 863-2185
I am generally interested in human evolution with a specific emphasis on genetic variation within and between species, gene duplication, and the evolution of regulatory elements. I find that the application of knowledge from these genetic investigations to explain (or help explain) phenotypic observations in humans (especially related to reproduction and human health) to be particularly exciting. My current project in the Makova Lab focuses on childhood obesity.
sarah.png
O
NA

Monika Cechova (Michalovova)
Ph.D student, Program in Biology
biomonika@psu.edu
I enjoy bioinformatics as a tool to study evolution of sex chromosomes, transposable elements and gene expression. I am interested in topics covering RNASeq, variant calling, mapping, genome assembly and new sequencing technologies. 
monika.png
S
http://sites.psu.edu/biomonika/

Di (Bruce) Chen
Ph.D student, Intercollege Graduate Program in Genetics
di.bruce.chen@gmail.com
Born and raised up in Nanjing, China. Trained as a computational biologist and amateur violinist. Big fan of backpacking, sci-fi, and research. Current work leverages bioinformatics tools and statistics in comparative genomics, with special interest in the regional variation of mutation rate in human genome.
bruce.png
R
NA

Wilfried Guiblet
Ph.D student, Program in Integrative Bioscience - Bioinformatics and Genomics
wmg133@psu.edu
I am interested in the relationship between genomics and human physiology. My current project is to relate the different type of DNA polymerases and their variability to specific mutation patterns.
wilfried.png
Mc
NA

Bonnie K Higgins
Research Technician
bkg102@psu.edu
Mitochondrial change and inheritance patterns
bonnie.png
Mt, O
NA

Kateryna Makova, Ph.D. 
Professor, Department of Biology 	Director, Center for Medical Genomics
kmakova@bx.psu.edu	Office: (814) 863-1619
Kateryna is interested in genomics, evolution, and human genetics. Her lab studies mutations using both computational and experimental approaches. Additional topics of interest include sex chromosome evolution and genomics of childhood obesity.
kateryna.png
?
NA

Samarth Rangavittal
Ph.D student, Program in Integrative Bioscience - Bioinformatics and Genomics
szr165@psu.edu
I am a grad student in the Bioinformatics and Genomics program. My primary interest is in genome assembly of short and long read sequence data. I also spend time thinking about topics like the sex chromosomes of primates, and how to mix computer science with biology in new and interesting ways.
samarth.png
S
NA

Nick Stoler
Ph.D student, Program in Integrative Bioscience - Bioinformatics and Genomics - Co-advised by Anton Nekrutenko
nicholas.stoler@gmail.com
My focus is on variation in mitochondrial DNA. I develop methods and software tools to analyze mitochondrial sequencing data, as well as data from other non-diploid systems like viral populations, cancer, and somatic variation. More information at https://nstoler.com https://github.com/NickSto/
nick.png
Mt
NA

Marta Tomaszkiewicz
Postdoctoral Scholar
marta@bx.psu.edu	Lab : (814) 863-2185
From the very beginning of my studies, I have been trying to understand the evolution of sex chromosomes across invertebrate and vertebrate taxa. Now, my postdoctoral research is focused on the Y chromosomes of primates. Using comparative genomics tools I am trying to answer important questions linked to the male heterogamety of our closest relatives.
marta.png
S
NA

Danling Ye
Undergraduate Student, Program in Veterinary and Biomedical Sciences
dxy5063@psu.edu
I am an undergraduate student currently pursuing a degree in Veterinary and Biomedical Sciences. In the lab, I am helping with a project involving genetics on the Y chromosome. I am also interested in research about the genetics involved in reproduction.
danling.png
S
NA

Rahulsimham Vegesna
Ph.D student, Program in Integrative Bioscience - Bioinformatics and Genomics - Co-advised by Paul Medvedev
rxv923@psu.edu
I am interested in understanding the mechanism that give rise to complex genomic structural variants in cancer. I am studying the role of different DNA polymerases in the formation of different structural variations.
rahul.png
Ms
NA

Arslan Zaidi
Postdoctoral Scholar
saz5078@psu.edu
I am interested in studying both the proximal (developmental) and ultimate (evolutionary) causes of human genetic variation. I am currently examining the transmission of mitochondrial heteroplasmies in families to understand the biological processes governing mitochondrial DNA inheritance in humans. When I’m not science-ing, I’m drawing or re-reading The Lord of the Rings. 
Arslan.png
Mt
NA


"""

def people():
	lines = PEOPLE.splitlines()
	lines = map(lambda x: x.strip(), lines)
	lines = filter(None, lines)
	group = [iter(lines)] * PEOPLE_FIELD_COUNT
	return zip(*group)



RESEARCH_FIELD_COUNT = 6

#Title
#Picture
#Short Description
#Long Description
#Icon
#Color

RESEARCH = """


Regional variation in mutation rates
chiaromonte.png
Mutation rates vary at a fine scale - even within individual chromosomes. We are using statistical modeling to study such regional variation in mutation rates. Moreover, we are able to segment the genome into regions with high and low rates of different mutation types.
Mutation rates vary at a fine scale - even within individual chromosomes. We are using statistical modeling to study such regional variation in mutation rates. Our analysis of insertions vs. deletions revealed that they are formed, at least in part, by different molecular mechanisms and that, in addition to replication, recombination plays an important role in indel formation (Kvikstad et al. 2007). Employing wavelet transformation techniques, we demonstrated the importance of genomic scale for regional variation in mutation rates (Kvikstad et al. 2009). Using specialized statistical approaches, we have investigated the structure of rate co-variation for four mutation types (point mutations, insertions, deletions, and microsatellite mutations) and simultaneously exploring the associations with multiple genomic features (Ananda et al. 2011). Moreover, we were able to segment the genome into regions with high and low rates of different mutations (Kurupummulage Don, Ananda, et al. 2013). Collaborator: Francesca Chiaromonte.
R
primary

Mitochondrial mutation dynamics
afgan.png
With sequencing now possible at high depth, we aim to examine mutation process directly, by examining the DNA of children and their parents. Mitochondrial DNA (mtDNA) provides a good starting point for such an investigation due to its compact size and high mutation rate.
With sequencing now possible at high depth, we aim to examine mutation process directly, by examining the DNA of children and their parents. Mitochondrial DNA (mtDNA) provides a good starting point for such an investigation due to its compact size and high mutation rate. We have established an experimental and computational pipeline to study mtDNA mutations and heteroplasmy frequency shifts (Goto et al. 2011), therefore we are in great position to study the patterns and rates of mtDNA mutations at unprecedented detail. Our recent study suggested that mitochondrial DNA undergoes through a very small bottleneck during oogenesis and that the number of heteroplasmies in the offspring increases with maternal age at fertilization (Rebolledo Jaramillo, Su, et al. 2014). Collaborators: Anton Nekrutenko, Francesca Chiaromonte, Rasmus Nielsen (Figure : Afgan et al. 2011).
Mt
warning

Childhood obesity
img
Obesity is becoming the major epidemic of the 21st century, and we are studying this disease in an integrated way - from the perspective of genomics, microbiomics, and epigenomics. We are following a large group of newborn children until they are at least 3 years of age.
Obesity is becoming the major epidemic of the 21st century, and we are studying this disease in an integrated way - from the perspective of genomics, microbiomics, and epigenomics. We are following a large group of newborn children until they are at least 3 years of age, and will be finding genes linked to the development of obesity in some of them. This collaboration includes clinicians, statisticians, and social scientists to obtain a multifaceted picture of this important phenotype. Collaborators: Ian Paul, Francesca Chiaromonte.
O
danger

Microsatellite variation and evolution
kelkar.png
We have investigated microsatellite evolution in the framework of a life-cycle hypothesis, according to which microsatellites are born and expand into adulthood, until their degradation and death. Microsatellites are implicated in numerous diseases and used as markers in population genetics and forensics.
We have investigated microsatellite evolution in the framework of a life-cycle hypothesis, according to which microsatellites are born and expand into adulthood, until their degradation and death (Kelkar et al. 2011). Studies of microsatellite mutagenesis have significant utility, as microsatellites are implicated in numerous diseases and are popular markers in population genetics and forensics. To commence this project, we had to determine the minimal number of repeats constituting a microsatellite (Kelkar et al. 2010). Next, using sequenced primate genomes, we dissected the intricacies of the mutational processes leading to microsatellite birth and death (Kelkar et al. 2011). We also evaluated the contribution of transposable elements to microsatellite births and deaths, and described the mechanisms of births for many disease-causing microsatellites (Kelkar et al. 2011). We investigated how interruptions affect microsatellite mutability (Ananda et al. 2014) and developed a computational pipeline, STR-FM, to genotype microsatellites from next-generation sequencing data (Fungtammasan, Ananda, et al. 2015). (Figure: Kelkar et al. 2011). Collaborators: Kristin Eckert, Paul Medvedev, Francesca Chiaromonte.
Mc
success

Sex chromosome evolution
wilson_sayres.png
We are determining the gene repertoire and structure of the Y chromosome in several mammalian species. The Y chromosome has very high rates of mutation compared with other chromosomes in the mammalian genome due to a higher number of cell divisions in the male than female germ line.
Several lines of our recent research have had mammalian sex chromosome evolution as a common theme. We have been studying the genomic determinants of X chromosome inactivation (XCI; Carrel et al. 2006). By contrasting the evolution of genes on sex chromosomes in eutherians with that of their autosomal homologs in marsupials and monotremes, we were able to shed light on the earliest stages of eutherian sex chromosome evolution (Wilson and Makova 2009). We also recently generated a comprehensive list of pseudogenes on the human Y and studied what determines whether a gene will survive or die on this chromosome (Wilson Sayres and Makova 2013). To test whether selection or drift leads to loss of genes on chromosome Y, we sequenced and analyzed all X-degenerate Y chromosome genes in gorilla (Goto et al. 2009) and are currently characterizing such genes in other primates. (Figure : Wilson Sayres & Makova 2013). Collaborators: Laura Carrel, Paul Medvedev.
S
info

"""

def research():
	lines = RESEARCH.splitlines()
	lines = map(lambda x: x.strip(), lines)
	lines = filter(None, lines)
	group = [iter(lines)] * RESEARCH_FIELD_COUNT
	return zip(*group)



COLLABORATORS_FIELD_COUNT = 5

#Name
#Job
#Picture
#Research Project
#URL

COLLABORATORS = """

Laura Carrel
Department of Biochemistry and Molecular Biology - Penn State Medical School at Hershey
laura.png
S
https://profiles.psu.edu/profiles/display/113588

Francesca Chiaromonte
Department of Statistics - Pennsylvania State University
francesca.png
Mc, Mt, R, O
https://sites.psu.edu/chiaromonte/

Michael DeGiorgio
Department of Biology
michael.png
Mc
http://www.personal.psu.edu/mxd60/people.html

Francisco J. Diaz
Associate Professor of Reproductive Biology
francisco.png
Mt
http://animalscience.psu.edu/directory/fjd10

Kristin Eckert
Department of Pathology - Penn State Medical School at Hershey
kristin.png
Mc
http://www.huck.psu.edu/people/kae4

Malcolm Ferguson-Smith
Emeritus Professor of Pathology - University of Cambridge
malcolm.png
S
http://www.vet.cam.ac.uk/directory/fs

Mitchell M. Holland
Associate Professor, Biochemistry & Molecular Biology
holland.png
Mt
http://forensics.psu.edu/faculty-staff/mmh20

Paul Medvedev
Computer Science & Engineering - Biochemistry & Molecular Biology - Pennsylvania State University
paul.png
Ms, S
http://medvedevgroup.com/medvedev.html

Anton Nekrutenko
Professor of Biochemistry and Molecular Biology - Pennsylvania State University
anton.png
Mt
http://www.bx.psu.edu/~anton/labSite/

Rasmus Nielsen
Center for Theoretical Evolutionary Genomics - University of California at Berkeley
rasmus.png
Mt
http://cteg.berkeley.edu/nielsen.html

Ian Paul
Professor of Pediatrics and Public Health Sciences - Penn State Hershey
ian.png
O, Mt
http://www.womenshealthcoe.psu.edu/bio_paul.htm

Oliver Ryder 
San Diego Zoological Society
oliver.png
S
http://biology.ucsd.edu/faculty/ryder.html


"""

def collaborators():
	lines = COLLABORATORS.splitlines()
	lines = map(lambda x: x.strip(), lines)
	lines = filter(None, lines)
	group = [iter(lines)] * COLLABORATORS_FIELD_COUNT
	return zip(*group)




BLOG_FIELD_COUNT = 3

#Title
#Author
#Article

BLOG = """

Inheriting Your Advisor's Philosophy
Kateryna Makova - Nov 24th 2015
Makova_24_Nov_2015

Un oeuf with the French puns already !
Samarth Rangavittal - Jan 18th 2016
Rangavittal_18_Jan_2016

Collaboration: Just As with Any Relationship, Open Communication Is the Key to Success
Sarah Craig - Feb 29th 2016
Craig_29_Feb_2016

Treat statistics as a science, and not a recipe.
Marzia Cremona - April 10th 2016
Cremona_10_Apr_2016

"""

def blog():
	lines = BLOG.splitlines()
	lines = map(lambda x: x.strip(), lines)
	lines = filter(None, lines)
	group = [iter(lines)] *BLOG_FIELD_COUNT
	return zip(*group)
